gotham/src/handler/assets/mod.rs

```diff
@@ -841,6 +913,62 @@ mod tests {
         assert_eq!(response.read_body().unwrap(), expected_body);
     }
 
+    #[test]
+    fn assets_range_request() {
+        let root = PathBuf::from("resources/test/assets");
+        let file_name = "doc.html";
+        let mut file = File::open(root.join(file_name)).unwrap();
+        let file_len = file.metadata().unwrap().len();
+        let router = build_simple_router(|route| route.get("/*").to_dir(root));
+        let server = TestServer::new(router).unwrap();
+
+        let tests = [
+            (Some(1), Some(123456789), 1, file_len - 1),
+            (None, Some(5), file_len - 5, 5),
+            (Some(5), None, 5, file_len - 5),
+            (Some(5), Some(5), 5, 1),
+            (Some(6), Some(5), 0, 0),
+        ];
+
+        for (range_begin, range_end, range_start, range_len) in tests {
+            let range_header = format!(
+                "bytes={}-{}",
+                range_begin.map(|i| i.to_string()).unwrap_or("".to_string()),
+                range_end.map(|i| i.to_string()).unwrap_or("".to_string())
+            );
+            let response = server
+                .client()
+                .get(format!("http://localhost/{file_name}"))
+                .with_header(RANGE, HeaderValue::from_str(&range_header).unwrap())
+                .perform()
+                .unwrap();
+            if range_start == 0 && range_len == 0 {
+                assert_eq!(response.status(), StatusCode::RANGE_NOT_SATISFIABLE);
+                break;
+            }
+            std::io::Seek::seek(&mut file, SeekFrom::Start(range_start)).unwrap();
```

Comment:
This looks odd. Try importing `std::io::Seek` and then calling `file.seek(...)`.

Reply:
I have changed this implementation

Reply:
GitHub shows no change. Did you forget to push?

Reply:
You're right, my mistake

Reply:
Thanks